### PR TITLE
vim-patch:fd31be29b822

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/fd31be29b8220ee1cb0b3460c82f2634ae3cc370

Not 100% sure if the `editing.txt` change relies on an unmerged change.

### CODE REVIEW

**Review by seandewar — APPROVED** [review]
LGTM. editing.txt change looks fine also

File: runtime/pack/dist/opt/termdebug/plugin/termdebug.vim [hunk 1]
```diff
@@ -1344,6 +1344,15 @@ func s:HandleCursor(msg)
...
       call s:GotoSourcewinOrCreateIt()
       if expand('%:p') != fnamemodify(fname, ':p')
+        augroup Termdebug
+          " Always open a file read-only instead of showing the ATTENTION
+          " prompt, since we are unlikely to want to edit the file.
+          " The file may be changed but not saved, warn for that.
+          au SwapExists * echohl WarningMsg
+                \ | echo 'Warning: file is being edited elsewhere'
+                \ | echohl None
+                \ | let v:swapchoice = '0'
```
[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/pack/dist/opt/termdebug/plugin/termdebug.vim:1354]: yeah this doesn't work. it's also easy for the message to be immediately overwritten by e.g. `showmode` in Vim at least; `echomsg` + showing the actual file name would probably be better :eyes:

[the remark this answers is not in the document]
Comment (clason) [reply runtime/pack/dist/opt/termdebug/plugin/termdebug.vim:1354]: Do you have a concrete suggestion for improvement? (I don't use `termdebug` myself.) Or should I just revert the changes to this file?

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/pack/dist/opt/termdebug/plugin/termdebug.vim:1354]: I recommend changing `0` to `o` to get what appears to be the desired behaviour described in the comments above.

**Review by seandewar — APPROVED** [review]

**Pull request merged**